src/groups/mqb/mqbstat/mqbstat_queuestats.h

```diff
@@ -432,6 +531,190 @@ inline bmqst::StatContext* QueueStatsDomain::statContext()
     return d_statContext_mp.get();
 }
 
+template <>
+inline void
+QueueStatsDomain::onEvent<QueueStatsDomain::EventType::Enum::e_ACK>(
+    BSLS_ANNOTATION_UNUSED bsls::Types::Int64 value)
+{
+    BSLS_ASSERT_SAFE(d_statContext_mp && "initialize was not called");
+    d_statContext_mp->adjustValue(DomainQueueStats::e_STAT_ACK, 1);
+}
+
+template <>
+inline void
+QueueStatsDomain::onEvent<QueueStatsDomain::EventType::Enum::e_ACK_TIME>(
+    bsls::Types::Int64 value)
+{
+    BSLS_ASSERT_SAFE(d_statContext_mp && "initialize was not called");
+    d_statContext_mp->reportValue(DomainQueueStats::e_STAT_ACK_TIME, value);
+}
+
+template <>
+inline void
+QueueStatsDomain::onEvent<QueueStatsDomain::EventType::Enum::e_NACK>(
+    BSLS_ANNOTATION_UNUSED bsls::Types::Int64 value)
+{
+    BSLS_ASSERT_SAFE(d_statContext_mp && "initialize was not called");
+    d_statContext_mp->adjustValue(DomainQueueStats::e_STAT_NACK, 1);
```

Comment:
Let's keep a comment like before, since we don't use `value`
```suggestion
    // For NACK, we don't care about the bytes value
    d_statContext_mp->adjustValue(DomainQueueStats::e_STAT_NACK, 1);
```